cmd/argocd/commands/app.go

```diff
@@ -1649,6 +1649,10 @@ func formatConditionsSummary(app argoappv1.Application) string {
 			items = append(items, cndType)
 		}
 	}
+
+	// Sort the keys by name
+	sort.Strings(items)
```

Comment:
Why do we need this?

Reply:
Because there is a flaky test. Building a string based on a map is not a good idea because it is unpredictable